src/services/chris_integration.tsx

```diff
@@ -147,17 +148,15 @@ class ChrisIntegration {
     let client: any = await ChrisAPIClient.getClient();
     try {
       console.log(img.fname)
+
+      // PL-DIRCOPY
       const dircopyPlugin = (await client.getPlugins({ "name_exact": this.FS_PLUGIN })).getItems()[0];
       // const params = await dircopyPlugin.getPluginParameters();
       const data: DirCreateData = { "dir": img.fname };
       const dircopyPluginInstance: PluginInstance = await client.createPluginInstance(dircopyPlugin.data.id, data);
+      console.log("PL-DIRCOPY task sent into the task queue")
 
-      const dirCopyResult = await pollingBackend(dircopyPluginInstance);
-      if (dirCopyResult.error) {
-        return [dirCopyResult];
-      }
-
```

Comment:
A question: since you've removed the `pollingBackend` code, how come this step of the process is deleted (where they used to poll the dircopyPluginInstance? Was the value of the dirCopyResult not relevant, so we don't need to check it anymore?

Reply:
Great question @colinywu 

The `dirCopyResult` is not relevant anymore since the plugin tasks are chained now. If in a chain one of the plugin fails, the subsequent plugins in the chain would also fail. Thus, by checking the result of the last plugin, we can identify whether the analysis has finished without errors or not.